reports/performance/_outliers.qmd

```diff
@@ -380,9 +416,10 @@ What are the sales with the largest miss relative to town and price quartile?
 qnt_df <- assessment_pin %>%
   mutate(Town = ccao::town_convert(meta_township_code)) %>%
   filter(
-    Town %in% c("Calumet", "Thornton", "Bloom"),
+    meta_triad_code == run_triad_code,
```

Comment:
I aligned this with Michael's so it doesn't get missed and things are getting changed in this section.